src/Dom/Document/Filter/LibxmlCompatibility.php

```diff
@@ -61,10 +61,16 @@ public function beforeLoad($html)
             $this->options[Option::LIBXML_FLAGS] |= constant('LIBXML_HTML_NODEFDTD');
         }
 
+        /**
+         * This flag prevents removing the closing tags used in inline JavaScript variables.
+         */
+        if (defined('LIBXML_SCHEMA_CREATE')) {
+            $this->options[Option::LIBXML_FLAGS] |= constant('LIBXML_SCHEMA_CREATE');
```

Comment:
Wow, how did you discover that this constant had this effect? The [description of this constant](https://www.php.net/manual/en/libxml.constants.php#:~:text=libxml_schema_create) doesn't seem to indicate this behavior:

> Create default/fixed value nodes during XSD schema validation

Reply:
With the help of Google and Stack Overflow. :) A lot of people already had this issue with DOMDocument.